components/starter-kits/starter-kits.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Container from '../container/container';
+import Link from '../link/link';
+import Kits from './starter-kits-data.json';
+import './starter-kits-style';
+
+// NOTE: The table classes used in this component correspond to
+// those used in the markdown utility. Ideally we will soon create
+// a table component that both the markdown utility and this page
+// can use. This component could even use something like griddle
+// to allow sorting and such.
```

Comment:
I'd really like to hold off on anything table related until we get a better markdown solution in place. I think the best option by far is to pipe all markdown to components that contain clean, isolated logic. It just feels like we've added a lot of features at this point, many of them in a hacky way which have come back to bite us.